jbtcentraltarget/multiple/jbtcentral-multiple.target

```diff
@@ -21,7 +21,7 @@
 
     <location includeAllPlatforms="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
       <repository location="https://sevntu-checkstyle.github.io/sevntu.checkstyle/update-site/"/>
-      <unit id="com.github.sevntu.checkstyle.checks.feature.feature.group" version="1.44.0"/>
+      <unit id="com.github.sevntu.checkstyle.checks.feature.feature.group" version="1.44.1"/>
```

Comment:
Is that the cause of the conflict ?

Reply:
no, but when unit testing the central update install, noticed that checkstyle as a new version. 
hadn't split to a new issue.